server/public/model/config.go

```diff
@@ -4746,6 +4790,40 @@ func (o *Config) Sanitize(pluginManifests []*Manifest) {
 	o.PluginSettings.Sanitize(pluginManifests)
 }
 
+func SanitizeDataSource(driverName, dataSource string) (string, error) {
```

Comment:
Now that this is in the `model` package, I'd welcome some documentation on what this exactly sanitizes.